__tests__/app.test.js

```diff
@@ -234,11 +234,87 @@ describe.only("POST /api/articles/:article_id/comments", () => {
       body: "Loved this article!",
     };
     return request(app)
-      .post("/api/articles/invalid_id/comments")
+      .post("/api/articles/999999/comments")
       .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+  test("404:POST/api/articles/:article_id/comments returns 404 when given a username that doesnt exist", () => {
+    const newComment = {
+      username: "i_dont_exist",
+      body: "Loved this article!",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("User not found");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200:PATCH /api/articles/:article_id returns an updated article when incrementing votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article).toMatchObject({
+          article_id: 1,
+          votes: 101,
+        });
+      });
+  });
+  test("200:PATCH /api/articles/:article_id returns an updated article when decrementing votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -1 })
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article).toMatchObject({
+          article_id: 1,
+          votes: 99,
+        });
+      });
+  });
+  test("400: returns 400 when inc_votes is missing", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
```

Comment:
This is fine if you want `400 - Bad  Request` to be the response when `inc_votes` is missing from the request. 
One thing to consider here is that one day in the future you might wish to be able to _PATCH_ other properties such as _title_, _body_, _img_url_... If the user must always include `inc_votes` in their request that could be a problem later if you expand on the endpoint 
Another approach here could be that if `inc_votes` is missing, the API responds with `200 -OK` and the article is not changed. This way your API is more open to expansion 💡